components/rendering/tests/codeblock_hide_lines.rs

```diff
@@ -0,0 +1,67 @@
+use std::collections::HashMap;
+
+use tera::Tera;
+
+use config::Config;
+use front_matter::InsertAnchor;
+use rendering::{render_content, RenderContext};
+
+macro_rules! colored_html_line {
+    ( @no $s:expr ) => {{
```

Comment:
what does `no` means here?

Reply:
it means I copied the test from hl_lines and didn't read enough what it did

Reply:
Ah sorry, I completely forgot that code